lctool/run.py

```diff
@@ -37,6 +37,44 @@ def lcget():
                 nf.write(content.encode('utf-8'))
                 nf.write('\n')
 
+def lcrefresh():
```

Comment:
Yes, it can be done by using argparse (https://docs.python.org/2/howto/argparse.html). Thanks.